tests/test_grids.py

```diff
@@ -107,6 +107,41 @@ def test_avoid_repeated_grids():
     assert field_set.temp.grid.name == 'grid0py'
 
 
+@pytest.mark.parametrize('mode', ['scipy', 'jit'])
+def test_multigrids_pointer(mode):
+    lon_g0 = np.linspace(0, 1e4, 21, dtype=np.float32)
+    lat_g0 = np.linspace(0, 1000, 2, dtype=np.float32)
+    depth_g0 = np.zeros((lon_g0.size, lat_g0.size, 5), dtype=np.float32)
+
+    def bath_func(lon):
+        return lon / 1000. + 10
+    bath = bath_func(lon_g0)
+
+    for i in range(depth_g0.shape[0]):
+        for k in range(depth_g0.shape[2]):
+            depth_g0[i, :, k] = bath[i] * k / (depth_g0.shape[2]-1)
+
+    grid_0 = RectilinearSGrid('grid0py', lon_g0, lat_g0, depth=depth_g0)
+    grid_1 = RectilinearSGrid('grid0py', lon_g0, lat_g0, depth=depth_g0)
```

Comment:
Why is this second `grid_1` needed? Why not use `grid_0` for all `Fields`? Or is that one of the things you want to test?

Reply:
It's a test. I'll add a comment on that